lib/poker_hands/CompareHands.rb

```diff
@@ -1,5 +1,24 @@
+require_relative 'value'
+
 class CompareHands
   def call(hand1, hand2)
-    true
+    h1_val = retreive_value(hand1)
+    h2_val = retreive_value(hand2)
+  end
+
+  def retreive_value(hand)
+    value = [
+              Value.new.royal_flush?(hand),
+              Value.new.straight_flush?(hand),
+              Value.new.four_of_a_kind?(hand),
+              Value.new.full_house?(hand),
+              Value.new.flush?(hand),
+              Value.new.straight?(hand),
+              Value.new.three_of_a_kind?(hand),
+              Value.new.two_pair?(hand),
+              Value.new.pair?(hand),
+              Value.new.high_card?(hand)
+            ]
+    value.max
```

Comment:
We could skip the variable assignment here if we wanted
```
            [
              Value.new.royal_flush?(hand),
              Value.new.straight_flush?(hand),
              Value.new.four_of_a_kind?(hand),
              Value.new.full_house?(hand),
              Value.new.flush?(hand),
              Value.new.straight?(hand),
              Value.new.three_of_a_kind?(hand),
              Value.new.two_pair?(hand),
              Value.new.pair?(hand),
              Value.new.high_card?(hand)
            ].max
```

Reply:
I thought it might be better to be explicit by writing `value.max` because the `.max` as you have it written could very easily be glanced over.